apps/external-devices/device-descriptors/Nintendo_WiiMote.json

```diff
@@ -0,0 +1,16 @@
+{
+    "deviceVendor": "Nintendo",
+    "deviceName": "WiiMote",
```

Comment:
@rpavlik I think those should all be separate device descriptors. I don't have any of those devices in front of me, so I don't know how many analogs/buttons they each have. Do you have that information at hand?